Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -600,6 +600,11 @@ theorem support_update_ne_zero [DecidableEq α] (h : b ≠ 0) :
   congr; apply Subsingleton.elim
 #align finsupp.support_update_ne_zero Finsupp.support_update_ne_zero
 
+@[simp] theorem update_idem (f : α →₀ M) (a : α) (b c : M) :
```

Comment:
maybe `update_update` would be a more intuitive name.

Reply:
This matches `Function.update_idem`

Reply:
excellent reason to stay by this name!